src/ekat/util/ekat_catch_main.cpp

```diff
@@ -1,6 +1,6 @@
 #define CATCH_CONFIG_RUNNER
 
-#include "catch2/catch.hpp"
+#include "catch2/catch_all.hpp"
```

Comment:
Is this their "all-inclusive" header (corresponding to our old one)? Can we get away with including just a couple of headers?

Also, I _think_ the macro `CATCH_CONFIG_RUNNER` might not be needed with the new version. I remember trying catch as a lib, and being able to not use it. But I might just misremember...

Reply:
Yes, that's the all-inclusive header, and I didn't have to use anything else for EKAT to build and run its tests.